mcrouter/CarbonRouterInstance-inl.h

```diff
@@ -131,7 +133,11 @@ CarbonRouterInstance<RouterInfo>* CarbonRouterInstance<RouterInfo>::createRaw(
 
   // Deleter is only called if unique_ptr::get() returns non-null.
   auto deleter = [](CarbonRouterInstance<RouterInfo>* inst) {
+#ifdef MCROUTER_OSS_BUILD
+    FB_LOG_EVERY_MS(WARNING, 10000) << "Destroying CarbonRouterInstance";
+#else
     LOG_EVERY_MS(WARNING, 10000) << "Destroying CarbonRouterInstance";
```

Comment:
This logging is not important so could be remove entirely.

Reply:
Done in 2b8cbd46—the custom deleter will probably still be necessary though, since the `CarbonRouterInstance` destructor is private which precludes using std::default_deleter.

Reply:
Yes custom deleter is needed. Thanks for fixing the OSS build!